### PR TITLE
feat(yatai): init onboard of yatai on AWS

### PR DESCRIPTION
**Pull request by rauerhans**
## Summary

Adds the AWS artifact for [Yatai](https://github.com/bentoml/Yatai) together with both the [Yatai-Deployment](https://github.com/bentoml/yatai-deployment) and the [Yatai-Image-Builder](https://github.com/bentoml/yatai-image-builder) component.

## Checklist

- [x]  No images hosted from dockerhub
- [ ]  Are dashboards present to understand the health of the application.  There **must** be at least 1 of these
    - [ ]  all databases should have dashboards
    - [ ]  ideally also have at least cpu/mem utilization dashboards for webserver tier of the app
    - [ ]  you can use `plural from-grafana` to convert a grafana dashboard found via google to our CRD
- [ ]  Are scaling runbooks present
    - [ ]  all databases **must** have scaling runbooks
    
        still needs a runbook
    - [ ]  you can use the charts in `pluralsh/module-library` to accelerate this
- [ ]  do you need to add config overlays?
    - [ ]  inputing secrets
    - [ ]  configuring autoscaling
- [ ]  If there’s a web-facing component to the app, we need to support OIDC authentication and setting up private networks if no authentication option is viable

     does not support OIDC, admin pw needs to be set up after the first install

- [ ]  All major clouds must be supported
    - [ ]  Azure
    - [x]  AWS
    - [ ]  GCP

### CODE REVIEW

File: yatai/helm/yatai/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+apiVersion: v2
+name: yatai
+description: helm chart for yatai
+type: application
+version: 0.1.1
```
Comment (DavidSpek) [inline yatai/helm/yatai/Chart.yaml:5]: bump chart version to `0.1.2` so this can be merged.

File: yatai/terraform/aws/deps.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+apiVersion: plural.sh/v1alpha1
+kind: Dependencies
+metadata:
+  description: yatai aws setup
+  version: 0.1.0
```
Comment (DavidSpek) [inline yatai/terraform/aws/deps.yaml:5]: bump to `0.1.1` so this can be merged.

**Review by DavidSpek — CHANGES_REQUESTED** [review]
Overall looks good. Few small comments before we can merge.

File: yatai/helm/yatai/values.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      hosts:
+        - yatai.dev.plural.sh
+  serviceAccount:
+    create: true
+    name: yatai
+  s3:
+    endpoint: "s3.amazonaws.com"
+
+yatai-deployment:
+  serviceAccount:
```
[the remark this answers is not in the document]
Comment (rauerhans) [reply yatai/helm/yatai/values.yaml:36]: will make a note

**Review by DavidSpek — APPROVED** [review]
LGTM

**Pull request merged**